[Immich] Install fails at jpegli-empty-dht-marker.patch

### ✅ Have you read and understood the above guidelines?

yes

### 🔎 Did you run the script with verbose mode enabled?

Yes, verbose mode was enabled and the output is included below

### 📜 What is the name of the script you are using?

Immich

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/immich.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 13

### 🧱 Is this Proxmox host running arm64?

No

### 📈 Which Proxmox version are you on?

9.2.10

### 📝 Provide a clear and concise description of the issue.

Install keeps stopping at the same point in the script.

### 🔄 Steps to reproduce the issue.

Ran with advanced install twice and default install



### ❌ Paste the full error output (if available).

`
================================================================================
COMBINED INSTALLATION LOG - immich
Container ID: 105
Session ID: 6ce57c9a
Timestamp: 2026-08-20 13:23:50
================================================================================

================================================================================
PHASE 1: CONTAINER CREATION (Host)
================================================================================

================================================================================
[2026-08-20 13:14:30] CONTAINER SETTINGS - immich
================================================================================
[2026-08-20 13:14:30] Application: immich
[2026-08-20 13:14:30] PVE Version: 9.2.10 (Kernel: 7.0.14-12-pve)
[2026-08-20 13:14:30] Container ID: 105
[2026-08-20 13:14:30] Operating System: debian (13)
[2026-08-20 13:14:30] Container Type: Unprivileged
[2026-08-20 13:14:30] Disk Size: 20 GB
[2026-08-20 13:14:30] CPU Cores: 4
[2026-08-20 13:14:30] RAM Size: 6144 MiB
[2026-08-20 13:14:30] GPU Passthrough: Enabled
[2026-08-20 13:14:30] Session ID: 6ce57c9a
[2026-08-20 13:14:34] [OK] Storage data2 (Free: 3.7TB  Used: 3.7GB) [Template]
[2026-08-20 13:14:37] [OK] Storage data1 (Free: 414.6GB  Used: 50.5GB) [Container]
[2026-08-20 13:14:37] [INFO] Validating storage 'data1'
[2026-08-20 13:14:38] [OK] Storage 'data1' (btrfs) validated
[2026-08-20 13:14:38] [INFO] Validating template storage 'data2'
[2026-08-20 13:14:38] [OK] Template storage 'data2' validated
[2026-08-20 13:14:38] [INFO] Searching for template 'debian-13'
[2026-08-20 13:14:39] [OK] Template search completed
[2026-08-20 13:14:40] [OK] Template debian-13-standard_13.6-1_amd64.tar.zst [local]

[2026-08-20 13:14:41] [OK] Template debian-13-standard_13.6-1_amd64.tar.zst is present and valid.
[2026-08-20 13:14:42] [INFO] Creating LXC container
[2026-08-20 13:14:49] [OK] LXC Container 105 was successfully created.

--- pct create output ---
Creating filesystem with 5242880 4k blocks and 1310720 inodes
Filesystem UUID: 0f9f5fa8-215f-487c-9ed2-805dde02f915
Superblock backups stored on blocks: 
	32768, 98304, 163840, 229376, 294912, 819200, 884736, 1605632, 2654208, 
extracting archive '/mnt/data2/template/cache/debian-13-standard_13.6-1_amd64.tar.zst'
Total bytes read: 553512960 (528MiB, 379MiB/s)
Detected container architecture: amd64
Creating SSH host key 'ssh_host_rsa_key' - this may take some time ...
done: SHA256:dwGOn36YI3DA/GKzBZsK82zTrixIrtAPPva0dyqB5Ek root@immich
Creating SSH host key 'ssh_host_ed25519_key' - this may take some time ...
done: SHA256:jrjZlHyB+1xkXfSFZoC5dnhXNPodrQvnkiSpE+J+YSw root@immich
Creating SSH host key 'ssh_host_ecdsa_key' - this may take some time ...
done: SHA256:V3xovdMC4o/6D2LuOwYUq/n6pGlwOn8bEtFAmBhxmhw root@immich
--- end pct create output ---
[2026-08-20 13:14:49] Detected Intel GPU
[2026-08-20 13:14:49] [OK] Automatically configuring INTEL GPU passthrough
[2026-08-20 13:14:49] [OK] INTEL GPU passthrough configured (2 devices)
[2026-08-20 13:14:49] [INFO] Starting LXC Container
[2026-08-20 13:14:52] [OK] Started LXC Container
[2026-08-20 13:14:52] [INFO] Waiting for network in LXC container
[2026-08-20 13:14:56] [OK] Network in LXC is reachable (ping)
[2026-08-20 13:14:56] [INFO] Detecting and setting correct GPU group IDs
[2026-08-20 13:15:06] [OK] GPU passthrough configured (video:44, render:992)
[2026-08-20 13:15:06] [INFO] Customizing LXC Container
Get:1 http://deb.debian.org/debian trixie InRelease [140 kB]
Get:2 http://security.debian.org trixie-security InRelease [43.4 kB]
Get:3 http://deb.debian.org/debian trixie-updates InRelease [47.3 kB]
Get:4 http://security.debian.org trixie-security/main amd64 Packages [245 kB]
Get:5 http://deb.debian.org/debian trixie/contrib Translation-en [49.6 kB]
Get:6 http://deb.debian.org/debian trixie-updates/main amd64 Packages.diff/Index [4,995 B]
Get:7 http://security.debian.org trixie-security/main Translation-en [150 kB]
Get:8 http://deb.debian.org/debian trixie/main Translation-en [6,484 kB]
Get:9 http://deb.debian.org/debian trixie-updates/main Translation-en [2,496 B]
Get:10 http://deb.debian.org/debian trixie-updates/main amd64 Packages T-2026-07-18-0800.33-F-2026-07-18-0800.33.pdiff [4,307 B]
Get:10 http://deb.debian.org/debian trixie-updates/main amd64 Packages T-2026-07-18-0800.33-F-2026-07-18-0800.33.pdiff [4,307 B]
Fetched 7,173 kB in 1s (11.7 MB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
The following additional packages will be installed:
  dirmngr gnupg gnupg-l10n gnupg-utils gpg gpg-agent gpg-wks-client gpgconf
  gpgsm gpgv libassuan9 libatomic1 libbrotli1 libcurl4t64 libgcrypt20
  libglib2.0-0t64 libglib2.0-data libgpg-error-l10n libgpg-error0 libgpm2
  libjq1 libksba8 libldap-common libldap2 libnghttp3-9 libnpth0t64 libonig5
  librtmp1 libssh2-1t64 mailcap mc-data pinentry-curses shared-mime-info unzip
  xdg-user-dirs
Suggested packages:
  dbus-user-session pinentry-gnome3 tor gpg-wks-server parcimonie xloadimage
  scdaemon tpm2daemon rng-tools low-memory-monitor gpm antiword arc | arcanist
  arj binutils cabextract catdoc texlive-binaries clzip | lunzip | lzd | lzip
  | lziprecover | minilzip | pdlzip | plzip ctorrent dbview
  default-jdk-headless djview4 djvulibre-bin elinks epub-utils
  | ncbi-entrez-direct exif genisoimage gettext ghostscript glade gputils gv
  imagemagick info jlha-utils | lhasa kchmviewer libaspell-dev libbatik-java
  libchm-bin liblz4-tool libxml2-utils links2 links | w3m | lynx lyx mikmod
  mpg321 mplayer odt2txt | unoconv p7zip-full par2 poedit | potool
  poppler-utils procyon-decompiler python3-boto python3-tz rar rpm sox sqlite3
  timidity unace | unace-nonfree unalz unar unrar | unrar-free vorbis-tools
  wimtools wv xpdf | pdf-viewer xpmutils zip zstd pinentry-doc
The following NEW packages will be installed:
  curl dirmngr gnupg gnupg-l10n gnupg-utils gnupg2 gpg gpg-agent
  gpg-wks-client gpgconf gpgsm gpgv jq libassuan9 libatomic1 libbrotli1
  libcurl4t64 libgcrypt20 libglib2.0-0t64 libglib2.0-data libgpg-error-l10n
  libgpg-error0 libgpm2 libjq1 libksba8 libldap-common libldap2 libnghttp3-9
  libnpth0t64 libonig5 librtmp1 libssh2-1t64 mailcap mc mc-data
  pinentry-curses shared-mime-info sudo unzip xdg-user-dirs
0 upgraded, 40 newly installed, 0 to remove and 22 not upgraded.
Need to get 14.6 MB of archives.
After this operation, 56.4 MB of additional disk space will be used.
Get:1 http://security.debian.org trixie-security/main amd64 libssh2-1t64 amd64 1.11.1-1+deb13u1 [245 kB]
Get:2 http://deb.debian.org/debian trixie/main amd64 sudo amd64 1.9.16p2-3+deb13u2 [2,086 kB]
Get:3 http://security.debian.org trixie-security/main amd64 libgcrypt20 amd64 1.11.0-7+deb13u1 [843 kB]
Get:4 http://deb.debian.org/debian trixie/main amd64 libbrotli1 amd64 1.1.0-2+b7 [307 kB]
Get:5 http://security.debian.org trixie-security/main amd64 libjq1 amd64 1.7.1-6+deb13u3 [168 kB]
Get:6 http://security.debian.org trixie-security/main amd64 jq amd64 1.7.1-6+deb13u3 [78.8 kB]
Get:7 http://deb.debian.org/debian trixie/main amd64 libldap2 amd64 2.6.10+dfsg-1 [194 kB]
Get:8 http://security.debian.org trixie-security/main amd64 unzip amd64 6.0-29+deb13u1 [173 kB]
Get:9 http://deb.debian.org/debian trixie/main amd64 libnghttp3-9 amd64 1.8.0-1 [67.7 kB]
Get:10 http://deb.debian.org/debian trixie/main amd64 librtmp1 amd64 2.4+20151223.gitfa8646d.1-2+b5 [58.8 kB]
Get:11 http://deb.debian.org/debian trixie/main amd64 libcurl4t64 amd64 8.14.1-2+deb13u4 [392 kB]
Get:12 http://deb.debian.org/debian trixie/main amd64 curl amd64 8.14.1-2+deb13u4 [270 kB]
Get:13 http://deb.debian.org/debian trixie/main amd64 libgpg-error0 amd64 1.51-4 [82.1 kB]
Get:14 http://deb.debian.org/debian trixie/main amd64 libassuan9 amd64 3.0.2-2 [61.5 kB]
Get:15 http://deb.debian.org/debian trixie/main amd64 gpgconf amd64 2.4.7-21+deb13u1+b4 [129 kB]
Get:16 http://deb.debian.org/debian trixie/main amd64 libksba8 amd64 1.6.7-2+b1 [136 kB]
Get:17 http://deb.debian.org/debian trixie/main amd64 libnpth0t64 amd64 1.8-3 [23.2 kB]
Get:18 http://deb.debian.org/debian trixie/main amd64 dirmngr amd64 2.4.7-21+deb13u1+b4 [384 kB]
Get:19 http://deb.debian.org/debian trixie/main amd64 gnupg-l10n all 2.4.7-21+deb13u1 [749 kB]
Get:20 http://deb.debian.org/debian trixie/main amd64 gpg amd64 2.4.7-21+deb13u1+b4 [634 kB]
Get:21 http://deb.debian.org/debian trixie/main amd64 pinentry-curses amd64 1.3.1-2 [86.4 kB]
Get:22 http://deb.debian.org/debian trixie/main amd64 gpg-agent amd64 2.4.7-21+deb13u1+b4 [271 kB]
Get:23 http://deb.debian.org/debian trixie/main amd64 gpgsm amd64 2.4.7-21+deb13u1+b4 [276 kB]
Get:24 http://deb.debian.org/debian trixie/main amd64 gnupg all 2.4.7-21+deb13u1 [417 kB]
Get:25 http://deb.debian.org/debian trixie/main amd64 gnupg2 all 2.4.7-21+deb13u1 [16.3 kB]
Get:26 http://deb.debian.org/debian trixie/main amd64 gpg-wks-client amd64 2.4.7-21+deb13u1+b4 [109 kB]
Get:27 http://deb.debian.org/debian trixie/main amd64 gpgv amd64 2.4.7-21+deb13u1+b4 [241 kB]
Get:28 http://deb.debian.org/debian trixie/main amd64 libonig5 amd64 6.9.9-1+b1 [189 kB]
Get:29 http://deb.debian.org/debian trixie/main amd64 libatomic1 amd64 14.2.0-19 [9,308 B]
Get:30 http://deb.debian.org/debian trixie/main amd64 libglib2.0-0t64 amd64 2.84.4-3~deb13u3 [1,519 kB]
Get:31 http://deb.debian.org/debian trixie/main amd64 libglib2.0-data all 2.84.4-3~deb13u3 [1,287 kB]
Get:32 http://deb.debian.org/debian trixie/main amd64 libgpg-error-l10n all 1.51-4 [114 kB]
Get:33 http://deb.debian.org/debian trixie/main amd64 libgpm2 amd64 1.20.7-11+b2 [14.4 kB]
Get:34 http://deb.debian.org/debian trixie/main amd64 libldap-common all 2.6.10+dfsg-1 [35.1 kB]
Get:35 http://deb.debian.org/debian trixie/main amd64 mailcap all 3.74 [32.8 kB]
Get:36 http://deb.debian.org/debian trixie/main amd64 mc-data all 3:4.8.33-1+deb13u1 [1,341 kB]
Get:37 http://deb.debian.org/debian trixie/main amd64 mc amd64 3:4.8.33-1+deb13u1 [550 kB]
Get:38 http://deb.debian.org/debian trixie/main amd64 shared-mime-info amd64 2.4-5+b2 [760 kB]
Get:39 http://deb.debian.org/debian trixie/main amd64 xdg-user-dirs amd64 0.18-2 [53.2 kB]
Get:40 http://deb.debian.org/debian trixie/main amd64 gnupg-utils amd64 2.4.7-21+deb13u1+b4 [194 kB]
Fetched 14.6 MB in 0s (29.8 MB/s)
Selecting previously unselected package sudo.
(Reading database ... 
Preparing to unpack .../00-sudo_1.9.16p2-3+deb13u2_amd64.deb ...
Unpacking sudo (1.9.16p2-3+deb13u2) ...
Selecting previously unselected package libbrotli1:amd64.
Preparing to unpack .../01-libbrotli1_1.1.0-2+b7_amd64.deb ...
Unpacking libbrotli1:amd64 (1.1.0-2+b7) ...
Selecting previously unselected package libldap2:amd64.
Preparing to unpack .../02-libldap2_2.6.10+dfsg-1_amd64.deb ...
Unpacking libldap2:amd64 (2.6.10+dfsg-1) ...
Selecting previously unselected package libnghttp3-9:amd64.
Preparing to unpack .../03-libnghttp3-9_1.8.0-1_amd64.deb ...
Unpacking libnghttp3-9:amd64 (1.8.0-1) ...
Selecting previously unselected package librtmp1:amd64.
Preparing to unpack .../04-librtmp1_2.4+20151223.gitfa8646d.1-2+b5_amd64.deb ...
Unpacking librtmp1:amd64 (2.4+20151223.gitfa8646d.1-2+b5) ...
Selecting previously unselected package libssh2-1t64:amd64.
Preparing to unpack .../05-libssh2-1t64_1.11.1-1+deb13u1_amd64.deb ...
Unpacking libssh2-1t64:amd64 (1.11.1-1+deb13u1) ...
Selecting previously unselected package libcurl4t64:amd64.
Preparing to unpack .../06-libcurl4t64_8.14.1-2+deb13u4_amd64.deb ...
Unpacking libcurl4t64:amd64 (8.14.1-2+deb13u4) ...
Selecting previously unselected package curl.
Preparing to unpack .../07-curl_8.14.1-2+deb13u4_amd64.deb ...
Unpacking curl (8.14.1-2+deb13u4) ...
Selecting previously unselected package libgpg-error0:amd64.
Preparing to unpack .../08-libgpg-error0_1.51-4_amd64.deb ...
Unpacking libgpg-error0:amd64 (1.51-4) ...
Selecting previously unselected package libassuan9:amd64.
Preparing to unpack .../09-libassuan9_3.0.2-2_amd64.deb ...
Unpacking libassuan9:amd64 (3.0.2-2) ...
Selecting previously unselected package libgcrypt20:amd64.
Preparing to unpack .../10-libgcrypt20_1.11.0-7+deb13u1_amd64.deb ...
Unpacking libgcrypt20:amd64 (1.11.0-7+deb13u1) ...
Selecting previously unselected package gpgconf.
Preparing to unpack .../11-gpgconf_2.4.7-21+deb13u1+b4_amd64.deb ...
Unpacking gpgconf (2.4.7-21+deb13u1+b4) ...
Selecting previously unselected package libksba8:amd64.
Preparing to unpack .../12-libksba8_1.6.7-2+b1_amd64.deb ...
Unpacking libksba8:amd64 (1.6.7-2+b1) ...
Selecting previously unselected package libnpth0t64:amd64.
Preparing to unpack .../13-libnpth0t64_1.8-3_amd64.deb ...
Unpacking libnpth0t64:amd64 (1.8-3) ...
Selecting previously unselected package dirmngr.
Preparing to unpack .../14-dirmngr_2.4.7-21+deb13u1+b4_amd64.deb ...
Unpacking dirmngr (2.4.7-21+deb13u1+b4) ...
Selecting previously unselected package gnupg-l10n.
Preparing to unpack .../15-gnupg-l10n_2.4.7-21+deb13u1_all.deb ...
Unpacking gnupg-l10n (2.4.7-21+deb13u1) ...
Selecting previously unselected package gpg.
Preparing to unpack .../16-gpg_2.4.7-21+deb13u1+b4_amd64.deb ...
Unpacking gpg (2.4.7-21+deb13u1+b4) ...
Selecting previously unselected package pinentry-curses.
Preparing to unpack .../17-pinentry-curses_1.3.1-2_amd64.deb ...
Unpacking pinentry-curses (1.3.1-2) ...
Selecting previously unselected package gpg-agent.
Preparing to unpack .../18-gpg-agent_2.4.7-21+deb13u1+b4_amd64.deb ...
Unpacking gpg-agent (2.4.7-21+deb13u1+b4) ...
Selecting previously unselected package gpgsm.
Preparing to unpack .../19-gpgsm_2.4.7-21+deb13u1+b4_amd64.deb ...
Unpacking gpgsm (2.4.7-21+deb13u1+b4) ...
Selecting previously unselected package gnupg.
Preparing to unpack .../20-gnupg_2.4.7-21+deb13u1_all.deb ...
Unpacking gnupg (2.4.7-21+deb13u1) ...
Selecting previously unselected package gnupg2.
Preparing to unpack .../21-gnupg2_2.4.7-21+deb13u1_all.deb ...
Unpacking gnupg2 (2.4.7-21+deb13u1) ...
Selecting previously unselected package gpg-wks-client.
Preparing to unpack .../22-gpg-wks-client_2.4.7-21+deb13u1+b4_amd64.deb ...
Unpacking gpg-wks-client (2.4.7-21+deb13u1+b4) ...
Selecting previously unselected package gpgv.
Preparing to unpack .../23-gpgv_2.4.7-21+deb13u1+b4_amd64.deb ...
Unpacking gpgv (2.4.7-21+deb13u1+b4) ...
Selecting previously unselected package libonig5:amd64.
Preparing to unpack .../24-libonig5_6.9.9-1+b1_amd64.deb ...
Unpacking libonig5:amd64 (6.9.9-1+b1) ...
Selecting previously unselected package libjq1:amd64.
Preparing to unpack .../25-libjq1_1.7.1-6+deb13u3_amd64.deb ...
Unpacking libjq1:amd64 (1.7.1-6+deb13u3) ...
Selecting previously unselected package jq.
Preparing to unpack .../26-jq_1.7.1-6+deb13u3_amd64.deb ...
Unpacking jq (1.7.1-6+deb13u3) ...
Selecting previously unselected package libatomic1:amd64.
Preparing to unpack .../27-libatomic1_14.2.0-19_amd64.deb ...
Unpacking libatomic1:amd64 (14.2.0-19) ...
Selecting previously unselected package libglib2.0-0t64:amd64.
Preparing to unpack .../28-libglib2.0-0t64_2.84.4-3~deb13u3_amd64.deb ...
Unpacking libglib2.0-0t64:amd64 (2.84.4-3~deb13u3) ...
Selecting previously unselected package libglib2.0-data.
Preparing to unpack .../29-libglib2.0-data_2.84.4-3~deb13u3_all.deb ...
Unpacking libglib2.0-data (2.84.4-3~deb13u3) ...
Selecting previously unselected package libgpg-error-l10n.
Preparing to unpack .../30-libgpg-error-l10n_1.51-4_all.deb ...
Unpacking libgpg-error-l10n (1.51-4) ...
Selecting previously unselected package libgpm2:amd64.
Preparing to unpack .../31-libgpm2_1.20.7-11+b2_amd64.deb ...
Unpacking libgpm2:amd64 (1.20.7-11+b2) ...
Selecting previously unselected package libldap-common.
Preparing to unpack .../32-libldap-common_2.6.10+dfsg-1_all.deb ...
Unpacking libldap-common (2.6.10+dfsg-1) ...
Selecting previously unselected package mailcap.
Preparing to unpack .../33-mailcap_3.74_all.deb ...
Unpacking mailcap (3.74) ...
Selecting previously unselected package mc-data.
Preparing to unpack .../34-mc-data_3%3a4.8.33-1+deb13u1_all.deb ...
Unpacking mc-data (3:4.8.33-1+deb13u1) ...
Selecting previously unselected package mc.
Preparing to unpack .../35-mc_3%3a4.8.33-1+deb13u1_amd64.deb ...
Unpacking mc (3:4.8.33-1+deb13u1) ...
Selecting previously unselected package shared-mime-info.
Preparing to unpack .../36-shared-mime-info_2.4-5+b2_amd64.deb ...
Unpacking shared-mime-info (2.4-5+b2) ...
Selecting previously unselected package unzip.
Preparing to unpack .../37-unzip_6.0-29+deb13u1_amd64.deb ...
Unpacking unzip (6.0-29+deb13u1) ...
Selecting previously unselected package xdg-user-dirs.
Preparing to unpack .../38-xdg-user-dirs_0.18-2_amd64.deb ...
Unpacking xdg-user-dirs (0.18-2) ...
Selecting previously unselected package gnupg-utils.
Preparing to unpack .../39-gnupg-utils_2.4.7-21+deb13u1+b4_amd64.deb ...
Unpacking gnupg-utils (2.4.7-21+deb13u1+b4) ...
Setting up libnpth0t64:amd64 (1.8-3) ...
Setting up libgpm2:amd64 (1.20.7-11+b2) ...
Setting up libgpg-error0:amd64 (1.51-4) ...
Setting up xdg-user-dirs (0.18-2) ...
Setting up unzip (6.0-29+deb13u1) ...
Setting up libbrotli1:amd64 (1.1.0-2+b7) ...
Setting up libgcrypt20:amd64 (1.11.0-7+deb13u1) ...
Setting up libldap-common (2.6.10+dfsg-1) ...
Setting up libglib2.0-data (2.84.4-3~deb13u3) ...
Setting up gnupg-l10n (2.4.7-21+deb13u1) ...
Setting up librtmp1:amd64 (2.4+20151223.gitfa8646d.1-2+b5) ...
Setting up libatomic1:amd64 (14.2.0-19) ...
Setting up sudo (1.9.16p2-3+deb13u2) ...
Setting up mc-data (3:4.8.33-1+deb13u1) ...
Setting up libnghttp3-9:amd64 (1.8.0-1) ...
Setting up gpgv (2.4.7-21+deb13u1+b4) ...
Setting up libassuan9:amd64 (3.0.2-2) ...
Setting up gpgconf (2.4.7-21+deb13u1+b4) ...
Setting up libssh2-1t64:amd64 (1.11.1-1+deb13u1) ...
Setting up libgpg-error-l10n (1.51-4) ...
Setting up mailcap (3.74) ...
Setting up libldap2:amd64 (2.6.10+dfsg-1) ...
Setting up libonig5:amd64 (6.9.9-1+b1) ...
Setting up libksba8:amd64 (1.6.7-2+b1) ...
Setting up pinentry-curses (1.3.1-2) ...
Setting up gpg-agent (2.4.7-21+deb13u1+b4) ...
Created symlink '/etc/systemd/user/sockets.target.wants/gpg-agent-browser.socket' → '/usr/lib/systemd/user/gpg-agent-browser.socket'.
Created symlink '/etc/systemd/user/sockets.target.wants/gpg-agent-extra.socket' → '/usr/lib/systemd/user/gpg-agent-extra.socket'.
Created symlink '/etc/systemd/user/sockets.target.wants/gpg-agent-ssh.socket' → '/usr/lib/systemd/user/gpg-agent-ssh.socket'.
Created symlink '/etc/systemd/user/sockets.target.wants/gpg-agent.socket' → '/usr/lib/systemd/user/gpg-agent.socket'.
Setting up libcurl4t64:amd64 (8.14.1-2+deb13u4) ...
Setting up libjq1:amd64 (1.7.1-6+deb13u3) ...
Setting up gpgsm (2.4.7-21+deb13u1+b4) ...
Setting up dirmngr (2.4.7-21+deb13u1+b4) ...
Created symlink '/etc/systemd/user/sockets.target.wants/dirmngr.socket' → '/usr/lib/systemd/user/dirmngr.socket'.
Setting up libglib2.0-0t64:amd64 (2.84.4-3~deb13u3) ...
No schema files found: doing nothing.
Setting up shared-mime-info (2.4-5+b2) ...
Setting up jq (1.7.1-6+deb13u3) ...
Setting up curl (8.14.1-2+deb13u4) ...
Setting up gpg (2.4.7-21+deb13u1+b4) ...
Created symlink '/etc/systemd/user/sockets.target.wants/keyboxd.socket' → '/usr/lib/systemd/user/keyboxd.socket'.
Setting up gnupg-utils (2.4.7-21+deb13u1+b4) ...
Setting up mc (3:4.8.33-1+deb13u1) ...
update-alternatives: using /usr/bin/mcview to provide /usr/bin/view (view) in auto mode
Setting up gpg-wks-client (2.4.7-21+deb13u1+b4) ...
Setting up gnupg (2.4.7-21+deb13u1) ...
Setting up gnupg2 (2.4.7-21+deb13u1) ...
Processing triggers for man-db (2.13.1-1) ...
Processing triggers for libc-bin (2.41-12+deb13u3) ...
[2026-08-20 13:15:29] [OK] Customized LXC Container
[2026-08-20 13:19:16] [ERROR] Installation failed in container 105 (exit code: 128)
  ⏳ Reporting failure to telemetry...
    ✔️   Failure reported
[2026-08-20 13:19:17] Installation log: /tmp/immich-105-6ce57c9a.log
[2026-08-20 13:19:39] [INFO] Restarting installation...
[2026-08-20 13:19:42] [OK] Storage data2 (Free: 3.7TB  Used: 3.7GB) [Template]
[2026-08-20 13:19:48] [OK] Storage data1 (Free: 411.1GB  Used: 54.0GB) [Container]
[2026-08-20 13:19:48] [INFO] Validating storage 'data1'
[2026-08-20 13:19:49] [OK] Storage 'data1' (btrfs) validated
[2026-08-20 13:19:49] [INFO] Validating template storage 'data2'
[2026-08-20 13:19:49] [OK] Template storage 'data2' validated
[2026-08-20 13:19:49] [INFO] Searching for template 'debian-13'
[2026-08-20 13:19:50] [OK] Template search completed
[2026-08-20 13:19:51] [OK] Template debian-13-standard_13.6-1_amd64.tar.zst [local]

[2026-08-20 13:19:52] [OK] Template debian-13-standard_13.6-1_amd64.tar.zst is present and valid.
[2026-08-20 13:19:53] [INFO] Creating LXC container
[2026-08-20 13:19:59] [OK] LXC Container 105 was successfully created.

--- pct create output ---
Creating filesystem with 5242880 4k blocks and 1310720 inodes
Filesystem UUID: 958bb20e-2d05-437b-b5c0-be28fd5bb7cf
Superblock backups stored on blocks: 
	32768, 98304, 163840, 229376, 294912, 819200, 884736, 1605632, 2654208, 
extracting archive '/mnt/data2/template/cache/debian-13-standard_13.6-1_amd64.tar.zst'
Total bytes read: 553512960 (528MiB, 369MiB/s)
Detected container architecture: amd64
Creating SSH host key 'ssh_host_ed25519_key' - this may take some time ...
done: SHA256:IBhfTE2EgTzEJuXehYjD/4N6T4O1NC0J9ZtALe5/3so root@immich
Creating SSH host key 'ssh_host_ecdsa_key' - this may take some time ...
done: SHA256:05kHO/HF0Sqlp/tqvBtgFLXUF4vMl5Dt+F6TvjnDW0A root@immich
Creating SSH host key 'ssh_host_rsa_key' - this may take some time ...
done: SHA256:F5Z+bkTx0NfxusaiIxIum895VolQ4yqoVMiOCIOv9GQ root@immich
--- end pct create output ---
[2026-08-20 13:19:59] Detected Intel GPU
[2026-08-20 13:19:59] [OK] Automatically configuring INTEL GPU passthrough
[2026-08-20 13:19:59] [OK] INTEL GPU passthrough configured (2 devices)
[2026-08-20 13:19:59] [INFO] Starting LXC Container
[2026-08-20 13:20:03] [OK] Started LXC Container
[2026-08-20 13:20:03] [INFO] Waiting for network in LXC container
[2026-08-20 13:20:07] [OK] Network in LXC is reachable (ping)
[2026-08-20 13:20:07] [INFO] Detecting and setting correct GPU group IDs
[2026-08-20 13:20:16] [OK] GPU passthrough configured (video:44, render:992)
[2026-08-20 13:20:17] [INFO] Customizing LXC Container
Get:1 http://security.debian.org trixie-security InRelease [43.4 kB]
Get:2 http://deb.debian.org/debian trixie InRelease [140 kB]
Get:3 http://deb.debian.org/debian trixie-updates InRelease [47.3 kB]
Get:4 http://security.debian.org trixie-security/main amd64 Packages [245 kB]
Get:5 http://deb.debian.org/debian trixie/contrib Translation-en [49.6 kB]
Get:6 http://deb.debian.org/debian trixie-updates/main amd64 Packages.diff/Index [4,995 B]
Get:7 http://security.debian.org trixie-security/main Translation-en [150 kB]
Get:8 http://deb.debian.org/debian trixie/main Translation-en [6,484 kB]
Get:9 http://deb.debian.org/debian trixie-updates/main Translation-en [2,496 B]
Get:10 http://deb.debian.org/debian trixie-updates/main amd64 Packages T-2026-07-18-0800.33-F-2026-07-18-0800.33.pdiff [4,307 B]
Get:10 http://deb.debian.org/debian trixie-updates/main amd64 Packages T-2026-07-18-0800.33-F-2026-07-18-0800.33.pdiff [4,307 B]
Fetched 7,173 kB in 1s (11.8 MB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
The following additional packages will be installed:
  dirmngr gnupg gnupg-l10n gnupg-utils gpg gpg-agent gpg-wks-client gpgconf
  gpgsm gpgv libassuan9 libatomic1 libbrotli1 libcurl4t64 libgcrypt20
  libglib2.0-0t64 libglib2.0-data libgpg-error-l10n libgpg-error0 libgpm2
  libjq1 libksba8 libldap-common libldap2 libnghttp3-9 libnpth0t64 libonig5
  librtmp1 libssh2-1t64 mailcap mc-data pinentry-curses shared-mime-info unzip
  xdg-user-dirs
Suggested packages:
  dbus-user-session pinentry-gnome3 tor gpg-wks-server parcimonie xloadimage
  scdaemon tpm2daemon rng-tools low-memory-monitor gpm antiword arc | arcanist
  arj binutils cabextract catdoc texlive-binaries clzip | lunzip | lzd | lzip
  | lziprecover | minilzip | pdlzip | plzip ctorrent dbview
  default-jdk-headless djview4 djvulibre-bin elinks epub-utils
  | ncbi-entrez-direct exif genisoimage gettext ghostscript glade gputils gv
  imagemagick info jlha-utils | lhasa kchmviewer libaspell-dev libbatik-java
  libchm-bin liblz4-tool libxml2-utils links2 links | w3m | lynx lyx mikmod
  mpg321 mplayer odt2txt | unoconv p7zip-full par2 poedit | potool
  poppler-utils procyon-decompiler python3-boto python3-tz rar rpm sox sqlite3
  timidity unace | unace-nonfree unalz unar unrar | unrar-free vorbis-tools
  wimtools wv xpdf | pdf-viewer xpmutils zip zstd pinentry-doc
The following NEW packages will be installed:
  curl dirmngr gnupg gnupg-l10n gnupg-utils gnupg2 gpg gpg-agent
  gpg-wks-client gpgconf gpgsm gpgv jq libassuan9 libatomic1 libbrotli1
  libcurl4t64 libgcrypt20 libglib2.0-0t64 libglib2.0-data libgpg-error-l10n
  libgpg-error0 libgpm2 libjq1 libksba8 libldap-common libldap2 libnghttp3-9
  libnpth0t64 libonig5 librtmp1 libssh2-1t64 mailcap mc mc-data
  pinentry-curses shared-mime-info sudo unzip xdg-user-dirs
0 upgraded, 40 newly installed, 0 to remove and 22 not upgraded.
Need to get 14.6 MB of archives.
After this operation, 56.4 MB of additional disk space will be used.
Get:1 http://security.debian.org trixie-security/main amd64 libssh2-1t64 amd64 1.11.1-1+deb13u1 [245 kB]
Get:2 http://deb.debian.org/debian trixie/main amd64 sudo amd64 1.9.16p2-3+deb13u2 [2,086 kB]
Get:3 http://security.debian.org trixie-security/main amd64 libgcrypt20 amd64 1.11.0-7+deb13u1 [843 kB]
Get:4 http://deb.debian.org/debian trixie/main amd64 libbrotli1 amd64 1.1.0-2+b7 [307 kB]
Get:5 http://deb.debian.org/debian trixie/main amd64 libldap2 amd64 2.6.10+dfsg-1 [194 kB]
Get:6 http://deb.debian.org/debian trixie/main amd64 libnghttp3-9 amd64 1.8.0-1 [67.7 kB]
Get:7 http://deb.debian.org/debian trixie/main amd64 librtmp1 amd64 2.4+20151223.gitfa8646d.1-2+b5 [58.8 kB]
Get:8 http://deb.debian.org/debian trixie/main amd64 libcurl4t64 amd64 8.14.1-2+deb13u4 [392 kB]
Get:9 http://deb.debian.org/debian trixie/main amd64 curl amd64 8.14.1-2+deb13u4 [270 kB]
Get:10 http://deb.debian.org/debian trixie/main amd64 libgpg-error0 amd64 1.51-4 [82.1 kB]
Get:11 http://deb.debian.org/debian trixie/main amd64 libassuan9 amd64 3.0.2-2 [61.5 kB]
Get:12 http://security.debian.org trixie-security/main amd64 libjq1 amd64 1.7.1-6+deb13u3 [168 kB]
Get:13 http://security.debian.org trixie-security/main amd64 jq amd64 1.7.1-6+deb13u3 [78.8 kB]
Get:14 http://deb.debian.org/debian trixie/main amd64 gpgconf amd64 2.4.7-21+deb13u1+b4 [129 kB]
Get:15 http://security.debian.org trixie-security/main amd64 unzip amd64 6.0-29+deb13u1 [173 kB]
Get:16 http://deb.debian.org/debian trixie/main amd64 libksba8 amd64 1.6.7-2+b1 [136 kB]
Get:17 http://deb.debian.org/debian trixie/main amd64 libnpth0t64 amd64 1.8-3 [23.2 kB]
Get:18 http://deb.debian.org/debian trixie/main amd64 dirmngr amd64 2.4.7-21+deb13u1+b4 [384 kB]
Get:19 http://deb.debian.org/debian trixie/main amd64 gnupg-l10n all 2.4.7-21+deb13u1 [749 kB]
Get:20 http://deb.debian.org/debian trixie/main amd64 gpg amd64 2.4.7-21+deb13u1+b4 [634 kB]
Get:21 http://deb.debian.org/debian trixie/main amd64 pinentry-curses amd64 1.3.1-2 [86.4 kB]
Get:22 http://deb.debian.org/debian trixie/main amd64 gpg-agent amd64 2.4.7-21+deb13u1+b4 [271 kB]
Get:23 http://deb.debian.org/debian trixie/main amd64 gpgsm amd64 2.4.7-21+deb13u1+b4 [276 kB]
Get:24 http://deb.debian.org/debian trixie/main amd64 gnupg all 2.4.7-21+deb13u1 [417 kB]
Get:25 http://deb.debian.org/debian trixie/main amd64 gnupg2 all 2.4.7-21+deb13u1 [16.3 kB]
Get:26 http://deb.debian.org/debian trixie/main amd64 gpg-wks-client amd64 2.4.7-21+deb13u1+b4 [109 kB]
Get:27 http://deb.debian.org/debian trixie/main amd64 gpgv amd64 2.4.7-21+deb13u1+b4 [241 kB]
Get:28 http://deb.debian.org/debian trixie/main amd64 libonig5 amd64 6.9.9-1+b1 [189 kB]
Get:29 http://deb.debian.org/debian trixie/main amd64 libatomic1 amd64 14.2.0-19 [9,308 B]
Get:30 http://deb.debian.org/debian trixie/main amd64 libglib2.0-0t64 amd64 2.84.4-3~deb13u3 [1,519 kB]
Get:31 http://deb.debian.org/debian trixie/main amd64 libglib2.0-data all 2.84.4-3~deb13u3 [1,287 kB]
Get:32 http://deb.debian.org/debian trixie/main amd64 libgpg-error-l10n all 1.51-4 [114 kB]
Get:33 http://deb.debian.org/debian trixie/main amd64 libgpm2 amd64 1.20.7-11+b2 [14.4 kB]
Get:34 http://deb.debian.org/debian trixie/main amd64 libldap-common all 2.6.10+dfsg-1 [35.1 kB]
Get:35 http://deb.debian.org/debian trixie/main amd64 mailcap all 3.74 [32.8 kB]
Get:36 http://deb.debian.org/debian trixie/main amd64 mc-data all 3:4.8.33-1+deb13u1 [1,341 kB]
Get:37 http://deb.debian.org/debian trixie/main amd64 mc amd64 3:4.8.33-1+deb13u1 [550 kB]
Get:38 http://deb.debian.org/debian trixie/main amd64 shared-mime-info amd64 2.4-5+b2 [760 kB]
Get:39 http://deb.debian.org/debian trixie/main amd64 xdg-user-dirs amd64 0.18-2 [53.2 kB]
Get:40 http://deb.debian.org/debian trixie/main amd64 gnupg-utils amd64 2.4.7-21+deb13u1+b4 [194 kB]
Fetched 14.6 MB in 0s (33.7 MB/s)
Selecting previously unselected package sudo.
(Reading database ... 
Preparing to unpack .../00-sudo_1.9.16p2-3+deb13u2_amd64.deb ...
Unpacking sudo (1.9.16p2-3+deb13u2) ...
Selecting previously unselected package libbrotli1:amd64.
Preparing to unpack .../01-libbrotli1_1.1.0-2+b7_amd64.deb ...
Unpacking libbrotli1:amd64 (1.1.0-2+b7) ...
Selecting previously unselected package libldap2:amd64.
Preparing to unpack .../02-libldap2_2.6.10+dfsg-1_amd64.deb ...
Unpacking libldap2:amd64 (2.6.10+dfsg-1) ...
Selecting previously unselected package libnghttp3-9:amd64.
Preparing to unpack .../03-libnghttp3-9_1.8.0-1_amd64.deb ...
Unpacking libnghttp3-9:amd64 (1.8.0-1) ...
Selecting previously unselected package librtmp1:amd64.
Preparing to unpack .../04-librtmp1_2.4+20151223.gitfa8646d.1-2+b5_amd64.deb ...
Unpacking librtmp1:amd64 (2.4+20151223.gitfa8646d.1-2+b5) ...
Selecting previously unselected package libssh2-1t64:amd64.
Preparing to unpack .../05-libssh2-1t64_1.11.1-1+deb13u1_amd64.deb ...
Unpacking libssh2-1t64:amd64 (1.11.1-1+deb13u1) ...
Selecting previously unselected package libcurl4t64:amd64.
Preparing to unpack .../06-libcurl4t64_8.14.1-2+deb13u4_amd64.deb ...
Unpacking libcurl4t64:amd64 (8.14.1-2+deb13u4) ...
Selecting previously unselected package curl.
Preparing to unpack .../07-curl_8.14.1-2+deb13u4_amd64.deb ...
Unpacking curl (8.14.1-2+deb13u4) ...
Selecting previously unselected package libgpg-error0:amd64.
Preparing to unpack .../08-libgpg-error0_1.51-4_amd64.deb ...
Unpacking libgpg-error0:amd64 (1.51-4) ...
Selecting previously unselected package libassuan9:amd64.
Preparing to unpack .../09-libassuan9_3.0.2-2_amd64.deb ...
Unpacking libassuan9:amd64 (3.0.2-2) ...
Selecting previously unselected package libgcrypt20:amd64.
Preparing to unpack .../10-libgcrypt20_1.11.0-7+deb13u1_amd64.deb ...
Unpacking libgcrypt20:amd64 (1.11.0-7+deb13u1) ...
Selecting previously unselected package gpgconf.
Preparing to unpack .../11-gpgconf_2.4.7-21+deb13u1+b4_amd64.deb ...
Unpacking gpgconf (2.4.7-21+deb13u1+b4) ...
Selecting previously unselected package libksba8:amd64.
Preparing to unpack .../12-libksba8_1.6.7-2+b1_amd64.deb ...
Unpacking libksba8:amd64 (1.6.7-2+b1) ...
Selecting previously unselected package libnpth0t64:amd64.
Preparing to unpack .../13-libnpth0t64_1.8-3_amd64.deb ...
Unpacking libnpth0t64:amd64 (1.8-3) ...
Selecting previously unselected package dirmngr.
Preparing to unpack .../14-dirmngr_2.4.7-21+deb13u1+b4_amd64.deb ...
Unpacking dirmngr (2.4.7-21+deb13u1+b4) ...
Selecting previously unselected package gnupg-l10n.
Preparing to unpack .../15-gnupg-l10n_2.4.7-21+deb13u1_all.deb ...
Unpacking gnupg-l10n (2.4.7-21+deb13u1) ...
Selecting previously unselected package gpg.
Preparing to unpack .../16-gpg_2.4.7-21+deb13u1+b4_amd64.deb ...
Unpacking gpg (2.4.7-21+deb13u1+b4) ...
Selecting previously unselected package pinentry-curses.
Preparing to unpack .../17-pinentry-curses_1.3.1-2_amd64.deb ...
Unpacking pinentry-curses (1.3.1-2) ...
Selecting previously unselected package gpg-agent.
Preparing to unpack .../18-gpg-agent_2.4.7-21+deb13u1+b4_amd64.deb ...
Unpacking gpg-agent (2.4.7-21+deb13u1+b4) ...
Selecting previously unselected package gpgsm.
Preparing to unpack .../19-gpgsm_2.4.7-21+deb13u1+b4_amd64.deb ...
Unpacking gpgsm (2.4.7-21+deb13u1+b4) ...
Selecting previously unselected package gnupg.
Preparing to unpack .../20-gnupg_2.4.7-21+deb13u1_all.deb ...
Unpacking gnupg (2.4.7-21+deb13u1) ...
Selecting previously unselected package gnupg2.
Preparing to unpack .../21-gnupg2_2.4.7-21+deb13u1_all.deb ...
Unpacking gnupg2 (2.4.7-21+deb13u1) ...
Selecting previously unselected package gpg-wks-client.
Preparing to unpack .../22-gpg-wks-client_2.4.7-21+deb13u1+b4_amd64.deb ...
Unpacking gpg-wks-client (2.4.7-21+deb13u1+b4) ...
Selecting previously unselected package gpgv.
Preparing to unpack .../23-gpgv_2.4.7-21+deb13u1+b4_amd64.deb ...
Unpacking gpgv (2.4.7-21+deb13u1+b4) ...
Selecting previously unselected package libonig5:amd64.
Preparing to unpack .../24-libonig5_6.9.9-1+b1_amd64.deb ...
Unpacking libonig5:amd64 (6.9.9-1+b1) ...
Selecting previously unselected package libjq1:amd64.
Preparing to unpack .../25-libjq1_1.7.1-6+deb13u3_amd64.deb ...
Unpacking libjq1:amd64 (1.7.1-6+deb13u3) ...
Selecting previously unselected package jq.
Preparing to unpack .../26-jq_1.7.1-6+deb13u3_amd64.deb ...
Unpacking jq (1.7.1-6+deb13u3) ...
Selecting previously unselected package libatomic1:amd64.
Preparing to unpack .../27-libatomic1_14.2.0-19_amd64.deb ...
Unpacking libatomic1:amd64 (14.2.0-19) ...
Selecting previously unselected package libglib2.0-0t64:amd64.
Preparing to unpack .../28-libglib2.0-0t64_2.84.4-3~deb13u3_amd64.deb ...
Unpacking libglib2.0-0t64:amd64 (2.84.4-3~deb13u3) ...
Selecting previously unselected package libglib2.0-data.
Preparing to unpack .../29-libglib2.0-data_2.84.4-3~deb13u3_all.deb ...
Unpacking libglib2.0-data (2.84.4-3~deb13u3) ...
Selecting previously unselected package libgpg-error-l10n.
Preparing to unpack .../30-libgpg-error-l10n_1.51-4_all.deb ...
Unpacking libgpg-error-l10n (1.51-4) ...
Selecting previously unselected package libgpm2:amd64.
Preparing to unpack .../31-libgpm2_1.20.7-11+b2_amd64.deb ...
Unpacking libgpm2:amd64 (1.20.7-11+b2) ...
Selecting previously unselected package libldap-common.
Preparing to unpack .../32-libldap-common_2.6.10+dfsg-1_all.deb ...
Unpacking libldap-common (2.6.10+dfsg-1) ...
Selecting previously unselected package mailcap.
Preparing to unpack .../33-mailcap_3.74_all.deb ...
Unpacking mailcap (3.74) ...
Selecting previously unselected package mc-data.
Preparing to unpack .../34-mc-data_3%3a4.8.33-1+deb13u1_all.deb ...
Unpacking mc-data (3:4.8.33-1+deb13u1) ...
Selecting previously unselected package mc.
Preparing to unpack .../35-mc_3%3a4.8.33-1+deb13u1_amd64.deb ...
Unpacking mc (3:4.8.33-1+deb13u1) ...
Selecting previously unselected package shared-mime-info.
Preparing to unpack .../36-shared-mime-info_2.4-5+b2_amd64.deb ...
Unpacking shared-mime-info (2.4-5+b2) ...
Selecting previously unselected package unzip.
Preparing to unpack .../37-unzip_6.0-29+deb13u1_amd64.deb ...
Unpacking unzip (6.0-29+deb13u1) ...
Selecting previously unselected package xdg-user-dirs.
Preparing to unpack .../38-xdg-user-dirs_0.18-2_amd64.deb ...
Unpacking xdg-user-dirs (0.18-2) ...
Selecting previously unselected package gnupg-utils.
Preparing to unpack .../39-gnupg-utils_2.4.7-21+deb13u1+b4_amd64.deb ...
Unpacking gnupg-utils (2.4.7-21+deb13u1+b4) ...
Setting up libnpth0t64:amd64 (1.8-3) ...
Setting up libgpm2:amd64 (1.20.7-11+b2) ...
Setting up libgpg-error0:amd64 (1.51-4) ...
Setting up xdg-user-dirs (0.18-2) ...
Setting up unzip (6.0-29+deb13u1) ...
Setting up libbrotli1:amd64 (1.1.0-2+b7) ...
Setting up libgcrypt20:amd64 (1.11.0-7+deb13u1) ...
Setting up libldap-common (2.6.10+dfsg-1) ...
Setting up libglib2.0-data (2.84.4-3~deb13u3) ...
Setting up gnupg-l10n (2.4.7-21+deb13u1) ...
Setting up librtmp1:amd64 (2.4+20151223.gitfa8646d.1-2+b5) ...
Setting up libatomic1:amd64 (14.2.0-19) ...
Setting up sudo (1.9.16p2-3+deb13u2) ...
Setting up mc-data (3:4.8.33-1+deb13u1) ...
Setting up libnghttp3-9:amd64 (1.8.0-1) ...
Setting up gpgv (2.4.7-21+deb13u1+b4) ...
Setting up libassuan9:amd64 (3.0.2-2) ...
Setting up gpgconf (2.4.7-21+deb13u1+b4) ...
Setting up libssh2-1t64:amd64 (1.11.1-1+deb13u1) ...
Setting up libgpg-error-l10n (1.51-4) ...
Setting up mailcap (3.74) ...
Setting up libldap2:amd64 (2.6.10+dfsg-1) ...
Setting up libonig5:amd64 (6.9.9-1+b1) ...
Setting up libksba8:amd64 (1.6.7-2+b1) ...
Setting up pinentry-curses (1.3.1-2) ...
Setting up gpg-agent (2.4.7-21+deb13u1+b4) ...
Created symlink '/etc/systemd/user/sockets.target.wants/gpg-agent-browser.socket' → '/usr/lib/systemd/user/gpg-agent-browser.socket'.
Created symlink '/etc/systemd/user/sockets.target.wants/gpg-agent-extra.socket' → '/usr/lib/systemd/user/gpg-agent-extra.socket'.
Created symlink '/etc/systemd/user/sockets.target.wants/gpg-agent-ssh.socket' → '/usr/lib/systemd/user/gpg-agent-ssh.socket'.
Created symlink '/etc/systemd/user/sockets.target.wants/gpg-agent.socket' → '/usr/lib/systemd/user/gpg-agent.socket'.
Setting up libcurl4t64:amd64 (8.14.1-2+deb13u4) ...
Setting up libjq1:amd64 (1.7.1-6+deb13u3) ...
Setting up gpgsm (2.4.7-21+deb13u1+b4) ...
Setting up dirmngr (2.4.7-21+deb13u1+b4) ...
Created symlink '/etc/systemd/user/sockets.target.wants/dirmngr.socket' → '/usr/lib/systemd/user/dirmngr.socket'.
Setting up libglib2.0-0t64:amd64 (2.84.4-3~deb13u3) ...
No schema files found: doing nothing.
Setting up shared-mime-info (2.4-5+b2) ...
Setting up jq (1.7.1-6+deb13u3) ...
Setting up curl (8.14.1-2+deb13u4) ...
Setting up gpg (2.4.7-21+deb13u1+b4) ...
Created symlink '/etc/systemd/user/sockets.target.wants/keyboxd.socket' → '/usr/lib/systemd/user/keyboxd.socket'.
Setting up gnupg-utils (2.4.7-21+deb13u1+b4) ...
Setting up mc (3:4.8.33-1+deb13u1) ...
update-alternatives: using /usr/bin/mcview to provide /usr/bin/view (view) in auto mode
Setting up gpg-wks-client (2.4.7-21+deb13u1+b4) ...
Setting up gnupg (2.4.7-21+deb13u1) ...
Setting up gnupg2 (2.4.7-21+deb13u1) ...
Setting up gpg-agent (2.4.7-21+deb13u1+b4) ...
Created symlink '/etc/systemd/user/sockets.target.wants/gpg-agent-browser.socket' → '/usr/lib/systemd/user/gpg-agent-browser.socket'.
Created symlink '/etc/systemd/user/sockets.target.wants/gpg-agent-extra.socket' → '/usr/lib/systemd/user/gpg-agent-extra.socket'.
Created symlink '/etc/systemd/user/sockets.target.wants/gpg-agent-ssh.socket' → '/usr/lib/systemd/user/gpg-agent-ssh.socket'.
Created symlink '/etc/systemd/user/sockets.target.wants/gpg-agent.socket' → '/usr/lib/systemd/user/gpg-agent.socket'.
Setting up libcurl4t64:amd64 (8.14.1-2+deb13u4) ...
Setting up libjq1:amd64 (1.7.1-6+deb13u3) ...
Setting up gpgsm (2.4.7-21+deb13u1+b4) ...
Setting up dirmngr (2.4.7-21+deb13u1+b4) ...
Created symlink '/etc/systemd/user/sockets.target.wants/dirmngr.socket' → '/usr/lib/systemd/user/dirmngr.socket'.
Setting up libglib2.0-0t64:amd64 (2.84.4-3~deb13u3) ...
No schema files found: doing nothing.
Setting up shared-mime-info (2.4-5+b2) ...
Setting up jq (1.7.1-6+deb13u3) ...
Setting up curl (8.14.1-2+deb13u4) ...
Setting up gpg (2.4.7-21+deb13u1+b4) ...
Created symlink '/etc/systemd/user/sockets.target.wants/keyboxd.socket' → '/usr/lib/systemd/user/keyboxd.socket'.
Setting up gnupg-utils (2.4.7-21+deb13u1+b4) ...
Setting up mc (3:4.8.33-1+deb13u1) ...
update-alternatives: using /usr/bin/mcview to provide /usr/bin/view (view) in auto mode
Setting up gpg-wks-client (2.4.7-21+deb13u1+b4) ...
Setting up gnupg (2.4.7-21+deb13u1) ...
Setting up gnupg2 (2.4.7-21+deb13u1) ...
Processing triggers for man-db (2.13.1-1) ...
Processing triggers for libc-bin (2.41-12+deb13u3) ...
[2026-08-20 13:20:39] [OK] Customized LXC Container
[2026-08-20 13:23:50] [ERROR] Installation failed in container 105 (exit code: 128)

================================================================================
PHASE 2: APPLICATION INSTALLATION (Container)
================================================================================
[2026-08-20 13:20:40] [INFO] Setting up Container OS
[2026-08-20 13:20:40] [OK] Set up Container OS
[2026-08-20 13:20:40] [OK] Network Connected: 192.168.2.154 fd4c:8943:d683:0:be24:11ff:fe50:249e 
[2026-08-20 13:20:41] [OK] IPv4 Internet Connected
[2026-08-20 13:20:41] [ERROR] IPv6 Internet Not Connected
[2026-08-20 13:20:41] [OK] Git DNS: github.com:(✔️ ) raw.githubusercontent.com:(✔️ ) api.github.com:(✔️ ) git.community-scripts.org:(✔️ )
[2026-08-20 13:20:41] [INFO] Updating Container OS
[2026-08-20 13:20:53] [OK] Updated Container OS
[2026-08-20 13:21:11] [INFO] Installing dependencies
[2026-08-20 13:22:26] [OK] Dependencies Installed
[2026-08-20 13:22:26] [INFO] Installing Mise
[2026-08-20 13:22:31] [OK] Installed Mise
[2026-08-20 13:22:31] [INFO] Configuring Debian Testing Repo
[2026-08-20 13:22:34] [OK] Configured Debian Testing repo
[2026-08-20 13:22:34] [INFO] Installing packages from Debian Testing repo
[2026-08-20 13:22:36] [OK] Installed packages from Debian Testing repo
[2026-08-20 13:22:36] [INFO] Setup uv 0.12.5
[2026-08-20 13:22:37] [OK] Setup uv 0.12.5
[2026-08-20 13:22:37] [INFO] Setup PostgreSQL 16
[2026-08-20 13:22:58] [OK] Setup PostgreSQL 16
[2026-08-20 13:23:00] [INFO] Fetching GitHub release: VectorChord (1.1.1)
[2026-08-20 13:23:02] [OK] Deployed: VectorChord (1.1.1)
[2026-08-20 13:23:05] [INFO] Setting up PostgreSQL Database
[2026-08-20 13:23:05] [WARN] Granting SUPERUSER privilege (security risk!)
[2026-08-20 13:23:05] [OK] Set up PostgreSQL Database
[2026-08-20 13:23:05] [INFO] Installing GCC-13 (available as fallback compiler)
[2026-08-20 13:23:12] [OK] Installed GCC-13
[2026-08-20 13:23:12] [WARN] Compiling Custom Photo-processing Libraries (can take anywhere from 15min to 2h)
[2026-08-20 13:23:12] [INFO] (1/5) Compiling libjxl
[2026-08-20 13:23:47] [ERROR] in line 197: exit code 128 (Invalid argument to exit): while executing command  git apply "$BASE_DIR"/server/sources/libjxl-patches/jpegli-empty-dht-marker.patch
`

### 🖼️ Additional context (optional).

Add full output from verbose mode logfile. Tried again with cloudflare dns to bypass local technitium and same result.

I've run into that exact error right now as well. 

<details>
<summary>My verbose output</summary>
```
libmimalloc3 is already the newest version (3.4.5+ds-1).
libde265-dev is already the newest version (1.1.1-1).
Summary:
Upgrading: 0, Installing: 0, Removing: 0, Not Upgrading: 571
✔  Installed/upgraded Testing repo packages
💡  Checking for updates to custom image-processing libraries (recompile time: 2-15min per library)
Already up to date.
⏳  Recompiling libjxlCloning into '/opt/staging/image-source/libjxl'...
remote: Enumerating objects: 39444, done.
remote: Counting objects: 100% (362/362), done.
remote: Compressing objects: 100% (239/239), done.
remote: Total 39444 (delta 260), reused 123 (delta 123), pack-reused 39082 (from 4)
Receiving objects: 100% (39444/39444), 39.38 MiB | 27.17 MiB/s, done.
Resolving deltas: 100% (32302/32302), done.
HEAD is now at 332feb17 Bump version to 0.11.2
Submodule 'third_party/testdata' (https://github.com/libjxl/testdata) registered for path 'testdata'
Submodule 'third_party/brotli' (https://github.com/google/brotli) registered for path 'third_party/brotli'
Submodule 'third_party/googletest' (https://github.com/google/googletest) registered for path 'third_party/googletest'
Submodule 'third_party/highway' (https://github.com/google/highway) registered for path 'third_party/highway'
Submodule 'third_party/lcms' (https://github.com/mm2/Little-CMS) registered for path 'third_party/lcms'
Submodule 'third_party/libjpeg-turbo' (https://github.com/libjpeg-turbo/libjpeg-turbo.git) registered for path 'third_party/libjpeg-turbo'
Submodule 'third_party/libpng' (https://github.com/glennrp/libpng.git) registered for path 'third_party/libpng'
Submodule 'third_party/sjpeg' (https://github.com/webmproject/sjpeg.git) registered for path 'third_party/sjpeg'
Submodule 'third_party/skcms' (https://skia.googlesource.com/skcms) registered for path 'third_party/skcms'
Submodule 'third_party/zlib' (https://github.com/madler/zlib.git) registered for path 'third_party/zlib'
Cloning into '/opt/staging/image-source/libjxl/testdata'...
Cloning into '/opt/staging/image-source/libjxl/third_party/brotli'...
Cloning into '/opt/staging/image-source/libjxl/third_party/googletest'...
Cloning into '/opt/staging/image-source/libjxl/third_party/highway'...
Cloning into '/opt/staging/image-source/libjxl/third_party/lcms'...
Cloning into '/opt/staging/image-source/libjxl/third_party/libjpeg-turbo'...
Cloning into '/opt/staging/image-source/libjxl/third_party/libpng'...
Cloning into '/opt/staging/image-source/libjxl/third_party/sjpeg'...
Cloning into '/opt/staging/image-source/libjxl/third_party/skcms'...
Cloning into '/opt/staging/image-source/libjxl/third_party/zlib'...
remote: Total 0 (delta 0), reused 0 (delta 0), pack-reused 0 (from 0)
remote: Enumerating objects: 10, done.
remote: Counting objects: 100% (10/10), done.
remote: Compressing objects: 100% (6/6), done.
remote: Total 6 (delta 3), reused 1 (delta 0), pack-reused 0 (from 0)
Unpacking objects: 100% (6/6), 157.86 KiB | 2.13 MiB/s, done.
From https://github.com/libjxl/testdata
* branch            ff8d743aaba05b3014f17e5475e576242fa979fc -> FETCH_HEAD
Submodule path 'testdata': checked out 'ff8d743aaba05b3014f17e5475e576242fa979fc'
remote: Total 0 (delta 0), reused 0 (delta 0), pack-reused 0 (from 0)
remote: Enumerating objects: 435, done.
remote: Counting objects: 100% (435/435), done.
remote: Compressing objects: 100% (221/221), done.
remote: Total 236 (delta 166), reused 33 (delta 11), pack-reused 0 (from 0)
Receiving objects: 100% (236/236), 378.21 KiB | 734.00 KiB/s, done.
Resolving deltas: 100% (166/166), completed with 153 local objects.
From https://github.com/google/brotli
* branch            36533a866ed1ca4b75cf049f4521e4ec5fe24727 -> FETCH_HEAD
Submodule path 'third_party/brotli': checked out '36533a866ed1ca4b75cf049f4521e4ec5fe24727'
remote: Total 0 (delta 0), reused 0 (delta 0), pack-reused 0 (from 0)
remote: Enumerating objects: 220, done.
remote: Counting objects: 100% (220/220), done.
remote: Compressing objects: 100% (109/109), done.
remote: Total 112 (delta 98), reused 5 (delta 0), pack-reused 0 (from 0)
Receiving objects: 100% (112/112), 55.30 KiB | 6.14 MiB/s, done.
Resolving deltas: 100% (98/98), completed with 98 local objects.
From https://github.com/google/googletest
* branch            6910c9d9165801d8827d628cb72eb7ea9dd538c5 -> FETCH_HEAD
Submodule path 'third_party/googletest': checked out '6910c9d9165801d8827d628cb72eb7ea9dd538c5'
remote: Total 0 (delta 0), reused 0 (delta 0), pack-reused 0 (from 0)
remote: Enumerating objects: 423, done.
remote: Counting objects: 100% (423/423), done.
remote: Compressing objects: 100% (189/189), done.
remote: Total 216 (delta 183), reused 39 (delta 26), pack-reused 0 (from 0)
Receiving objects: 100% (216/216), 508.20 KiB | 5.65 MiB/s, done.
Resolving deltas: 100% (183/183), completed with 157 local objects.
From https://github.com/google/highway
* branch            457c891775a7397bdb0376bb1031e6e027af1c48 -> FETCH_HEAD
Submodule path 'third_party/highway': checked out '457c891775a7397bdb0376bb1031e6e027af1c48'
remote: Total 0 (delta 0), reused 0 (delta 0), pack-reused 0 (from 0)
remote: Enumerating objects: 314, done.
remote: Counting objects: 100% (314/314), done.
remote: Compressing objects: 100% (157/157), done.
remote: Total 177 (delta 135), reused 40 (delta 19), pack-reused 0 (from 0)
Receiving objects: 100% (177/177), 2.11 MiB | 10.29 MiB/s, done.
Resolving deltas: 100% (135/135), completed with 105 local objects.
From https://github.com/mm2/Little-CMS
* branch            5176347635785e53ee5cee92328f76fda766ecc6 -> FETCH_HEAD
Submodule path 'third_party/lcms': checked out '5176347635785e53ee5cee92328f76fda766ecc6'
remote: Total 0 (delta 0), reused 0 (delta 0), pack-reused 0 (from 0)
remote: Enumerating objects: 632, done.
remote: Counting objects: 100% (632/632), done.
remote: Compressing objects: 100% (414/414), done.
remote: Total 494 (delta 248), reused 124 (delta 69), pack-reused 0 (from 0)
Receiving objects: 100% (494/494), 1.57 MiB | 7.82 MiB/s, done.
Resolving deltas: 100% (248/248), completed with 104 local objects.
From https://github.com/libjpeg-turbo/libjpeg-turbo
* branch            8ecba3647edb6dd940463fedf38ca33a8e2a73d1 -> FETCH_HEAD
Submodule path 'third_party/libjpeg-turbo': checked out '8ecba3647edb6dd940463fedf38ca33a8e2a73d1'
remote: Total 0 (delta 0), reused 0 (delta 0), pack-reused 0 (from 0)
remote: Enumerating objects: 518, done.
remote: Counting objects: 100% (518/518), done.
remote: Compressing objects: 100% (222/222), done.
remote: Total 310 (delta 180), reused 141 (delta 75), pack-reused 0 (from 0)
Receiving objects: 100% (310/310), 686.92 KiB | 6.73 MiB/s, done.
Resolving deltas: 100% (180/180), completed with 104 local objects.
From https://github.com/glennrp/libpng
* branch            872555f4ba910252783af1507f9e7fe1653be252 -> FETCH_HEAD
Submodule path 'third_party/libpng': checked out '872555f4ba910252783af1507f9e7fe1653be252'
remote: Total 0 (delta 0), reused 0 (delta 0), pack-reused 0 (from 0)
remote: Enumerating objects: 45, done.
remote: Counting objects: 100% (45/45), done.
remote: Compressing objects: 100% (19/19), done.
remote: Total 23 (delta 19), reused 5 (delta 2), pack-reused 0 (from 0)
Unpacking objects: 100% (23/23), 4.86 KiB | 497.00 KiB/s, done.
From https://github.com/webmproject/sjpeg
* branch            94e0df6d0f8b44228de5be0ff35efb9f946a13c9 -> FETCH_HEAD
Submodule path 'third_party/sjpeg': checked out '94e0df6d0f8b44228de5be0ff35efb9f946a13c9'
remote: Total 0 (delta 0), reused 0 (delta 0)
remote: Counting objects: 66, done
remote: Finding sources: 100% (66/66)
remote: Total 66 (delta 17), reused 50 (delta 17)
Unpacking objects: 100% (66/66), 94.00 KiB | 3.62 MiB/s, done.
From https://skia.googlesource.com/skcms
* branch            b2e692629c1fb19342517d7fb61f1cf83d075492 -> FETCH_HEAD
Submodule path 'third_party/skcms': checked out 'b2e692629c1fb19342517d7fb61f1cf83d075492'
remote: Total 0 (delta 0), reused 0 (delta 0), pack-reused 0 (from 0)
remote: Enumerating objects: 339, done.
remote: Counting objects: 100% (339/339), done.
remote: Compressing objects: 100% (157/157), done.
remote: Total 208 (delta 152), reused 74 (delta 48), pack-reused 0 (from 0)
Receiving objects: 100% (208/208), 118.49 KiB | 1011.00 KiB/s, done.
Resolving deltas: 100% (152/152), completed with 103 local objects.
From https://github.com/madler/zlib
* branch            51b7f2abdade71cd9bb0e7a373ef2610ec6f9daf -> FETCH_HEAD
Submodule path 'third_party/zlib': checked out '51b7f2abdade71cd9bb0e7a373ef2610ec6f9daf'
error: can't open patch '/opt/staging/base-images/server/sources/libjxl-patches/jpegli-empty-dht-marker.patch': No such file or directory
✖  in line 396: exit code 128 (Invalid argument to exit): while executing command  git apply "$BASE_DIR"/server/sources/libjxl-patches/jpegli-empty-dht-marker.patch

--- Last 20 lines of log ---
[2026-08-20 18:50:51] [WARN] Storage is dangerously low (85% used on /boot)
[2026-08-20 18:50:53] [INFO] Installing/upgrading Testing repo packages
[2026-08-20 18:50:54] [OK] Installed/upgraded Testing repo packages
[2026-08-20 18:50:54] [WARN] Checking for updates to custom image-processing libraries (recompile time: 2-15min per library)
[2026-08-20 18:50:55] [INFO] Recompiling libjxl
[2026-08-20 18:51:33] [ERROR] in line 396: exit code 128 (Invalid argument to exit): while executing command  git apply "$BASE_DIR"/server/sources/libjxl-patches/jpegli-empty-dht-marker.patch
-----------------------------------
```
</details>

I'm having the same issue. Error output below:

error: can't open patch '/opt/staging/base-images/server/sources/libjxl-patches/jpegli-empty-dht-marker.patch': No such file or directory
  ✖️  in line 197: exit code 128 (Invalid argument to exit): while executing command  git apply "$BASE_DIR"/server/sources/libjxl-patches/jpegli-empty-dht-marker.patch

  --- Last 20 lines of log ---
[2026-08-20 12:08:10] [INFO] Installing packages from Debian Testing repo
[2026-08-20 12:08:17] [OK] Installed packages from Debian Testing repo
[2026-08-20 12:08:22] [INFO] Setup uv 0.12.5
[2026-08-20 12:08:35] [OK] Setup uv 0.12.5
[2026-08-20 12:08:35] [INFO] Setup PostgreSQL 16
[2026-08-20 12:08:44] [WARN] PGDG repository not available for trixie — falling back to distro-provided PostgreSQL packages
[2026-08-20 12:08:44] [WARN] Distro will install PostgreSQL 18 (not the requested 16).
[2026-08-20 12:08:44] [WARN] Any PostgreSQL extension packages (e.g. vchord, pgvector) must be built for PostgreSQL 18 on trixie.
[2026-08-20 12:08:44] [INFO] Setup PostgreSQL (distro package)
[2026-08-20 12:09:04] [OK] Setup PostgreSQL 17
[2026-08-20 12:09:15] [INFO] Fetching GitHub release: VectorChord (1.1.1)
[2026-08-20 12:09:27] [OK] Deployed: VectorChord (1.1.1)
[2026-08-20 12:09:30] [INFO] Setting up PostgreSQL Database
[2026-08-20 12:09:30] [WARN] Granting SUPERUSER privilege (security risk!)
[2026-08-20 12:09:30] [OK] Set up PostgreSQL Database
[2026-08-20 12:09:30] [INFO] Installing GCC-13 (available as fallback compiler)
[2026-08-20 12:09:36] [OK] Installed GCC-13
[2026-08-20 12:09:36] [WARN] Compiling Custom Photo-processing Libraries (can take anywhere from 15min to 2h)
[2026-08-20 12:09:42] [INFO] (1/5) Compiling libjxl
[2026-08-20 12:13:20] [ERROR] in line 197: exit code 128 (Invalid argument to exit): while executing command  git apply "$BASE_DIR"/server/sources/libjxl-patches/jpegli-empty-dht-marker.patch



It fails at this point everytime.

> I'm having the same issue. Error output below...

Same here, identical failure
`[2026-08-20 15:58:19] [INFO] (1/5) Compiling libjxl
[2026-08-20 15:58:43] [ERROR] in line 197: exit code 128 (Invalid argument to exit): while executing command  git apply "$BASE_DIR"/server/sources/libjxl-patches/jpegli-empty-dht-marker.patch`

Looking at libjxl's own recent release notes, the empty-DHT-marker bug this patch exists to work around was fixed upstream in libjxl itself. It's likely that base-images (on main) removed the now-unnecessary patch file since newer libjxl doesn't need it.

--- a/ct/immich.sh
+++ b/ct/immich.sh
@@ -393,8 +393,8 @@ function compile_libjxl() {
     cd "$SOURCE"
     $STD git reset --hard "$LIBJXL_REVISION"
     $STD git submodule update --init --recursive --depth 1 --recommend-shallow
-    $STD git apply "$BASE_DIR"/server/sources/libjxl-patches/jpegli-empty-dht-marker.patch
-    $STD git apply "$BASE_DIR"/server/sources/libjxl-patches/jpegli-icc-warning.patch
+    $STD git apply -3 "$BASE_DIR"/server/sources/jpegli-patches/jpegli-empty-dht-marker.patch
+    $STD git apply -3 "$BASE_DIR"/server/sources/jpegli-patches/jpegli-icc-warning.patch
     mkdir build
     cd build
     $STD cmake \

--- a/install/immich-install.sh
+++ b/install/immich-install.sh
@@ -194,8 +194,8 @@ $STD git clone https://github.com/libjxl/libjxl.git "$SOURCE"
 cd "$SOURCE"
 $STD git reset --hard "$LIBJXL_REVISION"
 $STD git submodule update --init --recursive --depth 1 --recommend-shallow
-$STD git apply "$BASE_DIR"/server/sources/libjxl-patches/jpegli-empty-dht-marker.patch
-$STD git apply "$BASE_DIR"/server/sources/libjxl-patches/jpegli-icc-warning.patch
+$STD git apply -3 "$BASE_DIR"/server/sources/jpegli-patches/jpegli-empty-dht-marker.patch
+$STD git apply -3 "$BASE_DIR"/server/sources/jpegli-patches/jpegli-icc-warning.patch
 mkdir build
 cd build
 $STD cmake \

